solved_exercises/dani/quadratic.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from cmath import sqrt
+
+
+def polynome(a, b, c):
+    """This function calculates values of a univariate quadratic equation
```

Comment:
Mathematically speaking you are solving a quadratic ecuation / polynome. The docstring deserves to be correct.
Also, you did fix the bug in the code, but not in the docstring.